### DISCUSSION TITLE
IOS app

### DISCUSSION THREAD

**Discussion by extendedzero** [community/community#13651] [Mobile]
Hi. 
When I use ![alt text](...) to put link to picture and path contains space (%20) - ios app does not display this picture.

**Comment by brianlovin** [comment]
Hey @extendedzero I'm noticing the same behavior here on the GitHub website:

<img width="925" alt="Screen Shot 2022-03-30 at 15 07 05@2x" src="https://user-images.githubusercontent.com/1923260/160939045-c8ce0d3e-7a9b-453c-a5ec-1df9a16a3b2e.png">

I wonder if that image path is considered invalid everywhere 🤔
